### PR TITLE
trickster picks and score screen

### PR DESCRIPTION
**Pull request by ZakMiller**

### CODE REVIEW

File: assets/elm/main.elm [hunk 1]
```diff
@@ -699,6 +743,21 @@ viewGame model =
...
+                    if isTrickster model then
+                        [ button myButtonModifiers [ onClick GuessTopic ] [ text "Guess Topic" ] ]
+                    else
+                        []
+
+                Check ->
+                    if isGameMaster model then
+                        [ text "Was the trickster's guess correct?"
+                        , button myButtonModifiers [ onClick <| Validate <| True ] [ text "Yes" ]
+                        , button myButtonModifiers [ onClick <| Validate <| False ] [ text "No" ]
```
Comment (SylvanSign) [inline assets/elm/main.elm]: These can just be `onClick <| Validate True` or `onClick <| Validate False`. The 2nd back pipe isn't needed

File: lib/fake_artist/table.ex [hunk 2]
```diff
@@ -259,15 +270,24 @@ defmodule FakeArtist.Table do
...
         Logger.info(fn -> "Game Over." end)
 
-        %{
-          state
-          | big_state: :end,
-            winner: get_winner(players)
-        }
+        if is_trickster_picked do
+          Logger.info(fn -> "Trickster was choosen, so now he must choose..." end)
+          %{state | little_state: :tricky}
```
Comment (SylvanSign) [inline lib/fake_artist/table.ex]: I love how this all sounds when you read it out loud

File: lib/fake_artist/table.ex [hunk 3]
```diff
@@ -259,15 +270,24 @@ defmodule FakeArtist.Table do
...
+        else
+          Logger.info(fn -> "Trickster wasn't choosen, so he wins!" end)
```
Comment (SylvanSign) [inline lib/fake_artist/table.ex]: choosen

File: lib/fake_artist/table.ex [hunk 4]
```diff
@@ -346,20 +398,20 @@ defmodule FakeArtist.Table do
     Enum.count(state.players, fn {_k, v} -> v.voted_for == nil end) - game_master_vote == 0
   end
 
-  @spec get_winner(map()) :: binary()
-  defp get_winner(players) do
+  @spec is_trickster_picked(map()) :: boolean()
+  defp is_trickster_picked(players) do
```
Comment (SylvanSign) [inline lib/fake_artist/table.ex]: btw Elixir allows `?` as part of function names. So you can make this `is_trickster_picked?` or `trickster_picked?` for extra flavor if you want.

In fact I think that's a typical convention to show that something returns a Boolean value

File: lib/fake_artist/table.ex (hunk 3 above)
Comment (ZakMiller) [reply lib/fake_artist/table.ex]: twas a test

File: lib/fake_artist/table.ex (hunk 4 above)
Comment (ZakMiller) [reply lib/fake_artist/table.ex]: Had forgotten about that!

**Pull request merged**